### PR TITLE
Tanzu Pages

### PR DESCRIPTION
**Pull request by BrianMMcClain**

### CODE REVIEW

**Comment by jesse-martinez** [comment]
on the TAP page, we need to figure out why lightbox isn't working on the videos. and also why the boxes are in the hero aren't matching heights

Screen Shot 2021-12-16 at 6 40 17 AM

**Comment by jesse-martinez** [comment]
Light theme should also be addressed (:

File: layouts/tanzu/single.html [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+                    {{ if not .Site.Params.ui.breadcrumb_disable }}{{ partial "breadcrumb.html" . }}{{ end }}
+                    <div class="row align-items-center py-lg-5">
+                        <div class="col-12 col-lg-6 py-2 py-lg-5 pl-lg-0">
+                            <h1>{{ .Title }}</h1>
+                            <p class="lead">{{ .Description }}</p>
+                        </div>
+                        <div class="col-12 col-lg-6 py-2 py-lg-5 pr-lg-0 position-relative">
+                            <img class="product-icon position-absolute" alt="" src='{{ printf "%s%s%s" "/images/icons/" .Params.icon ".svg" | relURL }}'>
+                            <!-- Link Boxes -->
+                            <!-- TODO: Proper CSS fix for make link boxes the same size -->
```
Comment (BrianMMcClain) [inline layouts/tanzu/single.html:26]: I've put a hacky fix here which seems to work, but I don't believe is 100% the correct fix. Marked with a TODO to revisit when time allows

File: assets/scss/light-theme.scss [hunk 2]
```diff
@@ -368,6 +368,10 @@ div.aside div.aside-content {
   background: #D9E4EA !important;
 }
 
+#hero .product-links > .linked-box {
+  background: #D9E4EA !important;
+}
+
```
Comment (BrianMMcClain) [inline assets/scss/light-theme.scss:374]: This was the only thing that I found that wasn't rendering well in light mode, though I could have missed something.

**Pull request merged**